task2.ts

```diff
@@ -0,0 +1,65 @@
+interface Person {
+  name: string;
+  age: number;
+  email: string;
+}
+
+class Employee implements Person {
+  name: string;
```

Comment:
no need to add those as typescript will create the variables for you from the constructor, just do
```suggestion
  constructor(name: string, age: number, email: string) {}
```